decision-tree.yaml

```diff
@@ -278,12 +281,12 @@ questions:
         labels:
           - "verboden AI"
         redirects:
+          - nextConclusionId: "14.0.0"
```

Comment:
could you put this back in between 13.0.0 and 15.0.0 for clearer overview?

Reply:
This is unfortunately not possible, because otherwise if someone has both labels, it will be directed to the wrong conclusion as it also applies to the other conditions. We can perhaps solve this in the jexl logic?